share/namespace.go

```diff
@@ -203,6 +255,52 @@ func (n Namespace) Compare(n2 Namespace) int {
 	return bytes.Compare(n.data, n2.data)
 }
 
+// AddInt adds arbitrary int value to namespace, treating namespace as big-endian
+// implementation of int. It could be helpful for users to create an adjacent namespaces.
```

Comment:
```suggestion
// implementation of int. It could be helpful for users to create adjacent namespaces.
```

Reply:
We could also rename this method to indicate that's the usecase for it but i dont care that much